packages/@coorpacademy-components/src/organism/review-congrats/index.js

```diff
@@ -46,12 +46,18 @@ const ReviewCongrats = props => {
       <div className={style.containerCongrats}>
         <div className={style.title}>{title}</div>
         <div ref={container} className={style.containerCards}>
-          <MoleculeReviewCardCongrats {...cardCongratsStar} className={style.cardStar} />
-          <MoleculeReviewCardCongrats
-            {...cardCongratsRank}
-            timerAnimation={1800}
-            className={style.cardRank}
-          />
+          {cardCongratsRank ? (
+            <>
+              <MoleculeReviewCardCongrats {...cardCongratsStar} className={style.cardStar} />
+              <MoleculeReviewCardCongrats
+                {...cardCongratsRank}
+                timerAnimation={1800}
+                className={style.cardRank}
+              />
+            </>
```

Comment:
Je suis partant plutôt pour la syntaxe de tableau (la syntaxe Fragment n'est pas trop explicit pour montrer qu'il s'agit seulement de un sibling node)

```suggestion
            [
              <MoleculeReviewCardCongrats {...cardCongratsStar} className={style.cardStar} />
              ,
              <MoleculeReviewCardCongrats
                {...cardCongratsRank}
                timerAnimation={1800}
                className={style.cardRank}
              />
            ]
```